Kinvey.Core/Store/RemoveRequest.cs

```diff
@@ -79,6 +79,7 @@ public override async Task<KinveyDeleteResponse> ExecuteAsync()
                     break;
 
                 case WritePolicy.NETWORK_THEN_LOCAL:
+                case WritePolicy.LOCAL_THEN_NETWORK:
```

Comment:
@vinaygahlawat 
Yes, you are right. Local is the first, network is the second. 
When you are saying to deprecate, what you mean. Is it to use the "obsolete" attribute or to remove from our system at all.

Reply:
Yes, that is correct. In the case that a developer is using the `WritePolicy.NETWORK_THEN_LOCAL` enum, let's use `Obsolete` to deprecate this and direct them to use `WritePolicy.LOCAL_THEN_NETWORK`. This will give them time to change their code, which should be easy because the implementation is the same.

Reply:
@vinaygahlawat Ok!